ansible/roles/iroha-docker/defaults/main.yml

```diff
@@ -100,3 +92,16 @@ iroha_docker_logging_max_file: 3
 ## Iroha RAM limit
 # Maximum amount of RAM, it only limits the docker container (e.g. '1024m')
 iroha_docker_mem_limit: ''
+
+##  Iroha logs
+# Enable debug log
+iroha_log_debug: Flase
+
+## Iroha service
+# Iroha commands run on host which has 'iroha_service_host' variable set to true
```

Comment:
```suggestion
# Iroha commands run on host which has 'iroha_service_host' variable set to true
# You can set this variable to any node that you have 
```

Reply:
Also can you add comment that this setting is not required if you do not change the number of nodes ?